pmd-swift/src/test/resources/net/sourceforge/pmd/lang/swift/cpd/testdata/Swift5.9.swift

```diff
@@ -0,0 +1,37 @@
+import SwiftUI
```

Comment:
this file is no longer being parsed from `SwiftTokenizerTest`… the test method seems to have been removed

Reply:
the path to the SwiftTokenizerTest changed, I've added it back